lib/dependencies.js

```diff
@@ -143,7 +147,12 @@ module.exports = class Dependencies {
     for (let i = 0; i < dependencies.length; i++) {
       let depPath = path.normalize(dependencies[i]);
       if (!path.isAbsolute(depPath)) {
-        depPath = path.resolve(this.rootDir, fileDir, depPath);
+        for (let index = 0; index < this.rootDirs.length; index++) {
+          let tempPath = path.resolve(this.rootDirs[index], fileDir, depPath);
+          if (fs.existsSync(tempPath)) {
+            depPath = tempPath;
```

Comment:
This code seems weird to me. I don't see where the mapped paths from the fsmerger are taken into account. Also, there was a definitive file that was already accessed -- I don't think we should go looking for it again at this point. Instead, I think I'd rather require that `filePath` is absolute now and just throw an error if passed a relative path.

Reply:
After conversation with @chriseppstein offline, setDependencies now must accept the absolute path. This will be a breaking change. And we will bump the major version for this change.